src/platform.ts

```diff
@@ -55,3 +55,7 @@ function isAlpineLinux(): boolean {
 export function getSupportedOS(): NodeJS.Platform | 'alpine' {
   return isAlpineLinux() ? 'alpine' : process.platform;
 }
+
+export function shellIsRequired() {
+  return getSupportedOS() === 'win32';
+}
```

Comment:
I would say that now is probably a good time to consolidate the code to detect linux + alpine + windows, wdyt?

So I would:
- Rename this to `isWindows()` and move it to `platform.ts`
- Change the code to `return process.platform.startsWith("win")`
- Modify `scanner-cli.ts @ normalizePlatformName()` to use `isWindows()`, `isLinux()`
- Use `isWindows()` instead of `isShellRequired()` here

This is a very opinionated take, feel free to challenge, wdyt?